tests/std/tests/GH_004657_expected_constraints_permissive/test.compile.pass.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <expected>
+#include <locale>

```

Comment:
Oh, I think `<locale>` should be here. Because it provides some explicit template instantation definitions which triggered the original issue.
https://github.com/microsoft/STL/blob/8dc4faadafb52e3e0a627e046b41258032d9bc6a/stl/inc/xloctime#L1057-L1067